### PR TITLE
Upgrade foursight to run repliseq v16.1

### PR DESCRIPTION
**Pull request by clarabakker**

### CODE REVIEW

**Review by aschroed — APPROVED** [review]
I might suggest bumping the version to 14.1.0 as this is updating a feature rather than fixing a bug but I leave that up to you.

**Review by aschroed — CHANGES_REQUESTED** [review]
Looking again should not v16.1 be added to TSA-seq and NAD-seq  in wfr_utils?  Or am I not understanding how this works?

File: chalicelib_fourfront/checks/helpers/wfr_utils.py [hunk 1]
```diff
@@ -189,11 +189,13 @@
...
+                          'RepliSeq_Pipeline_v16_step1',
+                          'RepliSeq_Pipeline_v16.1_step1'],
     # OFFICIAL - 1 STEP
     'Multi-stage Repli-seq': ['RepliSeq_Pipeline_v13.1_step1',
                               'RepliSeq_Pipeline_v14_step1',
-                              'RepliSeq_Pipeline_v16_step1'],
+                              'RepliSeq_Pipeline_v16_step1',
+                              'RepliSeq_Pipeline_v16.1_step1'],
     # Preliminary - Released to network
     'NAD-seq':       ['RepliSeq_Pipeline_v13.1_step1', 'RepliSeq_Pipeline_v14_step1', 'RepliSeq_Pipeline_v16_step1'],
```
Comment (aschroed) [inline chalicelib_fourfront/checks/helpers/wfr_utils.py:200]: Should v16.1 be added here and in TSA-seq section above?

**Review by aschroed — APPROVED** [review]

File: chalicelib_fourfront/checks/helpers/wfr_utils.py (hunk 1 above)
Comment (clarabakker) [reply chalicelib_fourfront/checks/helpers/wfr_utils.py]: Good point!

**Pull request merged**